### PR TITLE
Update sprockets links to point to rails org

### PR DESCRIPTION
**Pull request by josh**
Sprockets and ExecJS now live under the Rails org.

/cc @rafaelfranca

### CODE REVIEW

File: guides/source/getting_started.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ Rails adds the `therubyracer` gem to the generated `Gemfile` in a
 commented line for new apps and you can uncomment if you need it.
 `therubyrhino` is the recommended runtime for JRuby users and is added by
 default to the `Gemfile` in apps generated under JRuby. You can investigate
-all the supported runtimes at [ExecJS](https://github.com/sstephenson/execjs#readme).
+all the supported runtimes at [ExecJS](https://github.com/josh/execjs#readme).
```
Comment (bensie) [inline guides/source/getting_started.md]: Should this be `/rails/execjs#readme`?

File: railties/lib/rails/generators/rails/plugin/templates/rails/javascripts.js [hunk 2]
```diff
@@ -7,7 +7,7 @@
 // It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
 // compiled file.
 //
-// Read Sprockets README (https://github.com/sstephenson/sprockets#sprockets-directives) for details
+// Read Sprockets README (https://github.com/josh/sprockets#sprockets-directives) for details
```
Comment (bensie) [inline railties/lib/rails/generators/rails/plugin/templates/rails/javascripts.js]: `rails/sprockets#sprockets-directives`

File: guides/source/getting_started.md (hunk 1 above)
Comment (josh) [inline guides/source/getting_started.md]: lol, i'm an idiot

**Comment by josh** [comment]
@bensie ha, good catch!

**Comment by senny** [comment]
@josh can you squash the commits? :hamster:

**Comment by rafaelfranca** [comment]
Merged at 99d0c2ac67e9b6593e00f2144e57ad81afba5468

**Comment by josh** [comment]
> @josh can you squash the commits? :hamster:

Why isn't that a github feature! :trollface:

**Pull request closed without merging**